### ISSUE TITLE
Add Arabic Translations

### ISSUE THREAD

**Issue by kevin-sheldon** [covid-alert-ny/covid-green-app#553]
Need to get app translations for Arabic. Spreadsheet required to send over to translators.

**Comment by AlanSl** [comment]
Hi @kevin-sheldon  no new spreadsheet is needed, the same spreadsheet here can be used for any new languages: https://github.com/covid-alert-ny/covid-green-app/issues/551#issuecomment-706280185

The translators can simply add a new column for each new language

**Comment by mboskin** [comment]
Please find the Arabic translation included in the attached.
 
Please note:
- Arabic has the same issue in Excel that Yiddish had before: in the cells that start with text in English or a symbol, the cell was set to read Left-to-Right automatically, but the Arabic still reads Right-to-Left. This doesn’t seem to have been an issue when exporting the Yiddish into the final app, but please make sure to keep it in mind in case extra steps were required for the Yiddish.
 
Let us know if any questions come up or if you would like the translators 
[covid-alert-ny-translations-v7_rev2.xlsx](https://github.com/covid-alert-ny/covid-green-app/files/5454050/covid-alert-ny-translations-v7_rev2.xlsx)

to review any screenshots once they upload the translations.

**Comment by AlanSl** [comment]
@mboskin or @kevin-sheldon Can you please forward the set of Arabic screenshots to the translators for review - thank you. 

[**Arabic screenshots.zip**](https://github.com/covid-alert-ny/covid-green-app/files/5483041/Arabic.screenshots.zip)

Also, are there currently any plans for any more languages (e.g. Urdu?)

**Comment by AlanSl** [comment]
Summarising [translator's feedback](https://nearform.slack.com/files/U019QN33A4A/F01EGU52F0R/arabic_screenshots_edits.docx):

- [x] Hidden text (nothing to do here - this is just the screenshot scroll position)
- [x] Fix reversed phone numbers everywhere
- [x] Remove 212-COVID19 style phone number everywhere
- [x] Number format for floating chart label - % at the end
- [x] Translate months
- [x] Fix date format on alerts to be like `2 نوفمبر، 2020`*
- [x] Fix pluralisation
- [x] Fix RTL direction of symptoms history date line*
- [x] Tour image text (not done - we don't translate these for any other languages)

*(these date formats appear to actually be correctly RTL, but it's a US-style month-first date translated, whereas the translator appears to be expecting a non-US-style day-first date. To change this we'd have to change every language)

**Comment by mboskin** [comment]
@AlanSl What do the checked boxes signify?

**Comment by AlanSl** [comment]
@mboskin I'm working through this among other things today. Checked means done. When they're all checked this ticket will be ready for the next build (aiming for end of today, got a few other small things to go in too).

**Comment by mboskin** [comment]
@AlanSl do you have a next build scheduled? Colm and I had emailed about the next release going into UAT Nov. 23.

**Comment by AlanSl** [comment]
The build I'm referring to will just be for internal testing as we did with Yiddish. It's mainly just so our own QA testers can check the latest small fixes. It doesn't affect the release schedule. 

So it's not anything formally scheduled and doesn't affect UAT dates. 

The above comment is mostly for my own reference and so there's a record here of what was fixed - apologies for any confusion caused.

**Comment by mboskin** [comment]
Thanks for the explanation

**Comment by AlanSl** [comment]
Arabic should be available and working for the entire app in test build 53, I'm assigning this for our internal testing.

**Comment by denisedresbach** [comment]
I have verified as much as I can by changing to Arabic language and viewing all screens, trying any links and bringing up confirmation messages and uploading/sharing IDs. Tested on both iOS and Android. 
1.1.0.53. I have written up a few tickets where issues were found and will close this main ticket as done.

Issues written up-
Onboarding with Android device language set to Arabic shows in English, so I couldn't test that. 
https://github.com/covid-alert-ny/covid-green-app/issues/574
Data Policy in English - https://github.com/covid-alert-ny/covid-green-app/issues/572
